### ISSUE TITLE
TODO:WORD is not getting picked up

### ISSUE THREAD

**Issue by chharvey** [mrodalgaard/atom-todo-show#196]
In some of my comments I have `TODO:WORD` keywords where "WORD" defines further granularity of todo keywords. E.g., `CHANGED:DEPRECATED`, `CHANGED:NEW`, `NB:LINK` etc.

These are not getting picked up by the Todo-Show tool. In order for them to be seen, I have to add a space after the colon (`CHANGED: DEPRECATED`) but I think it would be better if the first format were included.

**Comment by mrodalgaard** [comment]
You can include your own format to todo-show with a little fiddling of the `findUsingRegex` config. Or you can use the tagging format instead by using the following format of your todos: `TODO: This should be fixed #DEPRECATED`.

**Comment by chharvey** [comment]
Thanks for the Regex idea. I did some digging and found that instead of modifying my `findUsingRegex` config, I could take advantage of the default: namely, the `[\{\[\(]` option.

**So I think my solution would be to change `TODO:DEPRECATED` to `TODO{DEPRECATED}` and that would be sufficient to get picked up by this package.**

One more thing, I noticed the default regex can be simplified:

default:
```regex
/\b(${TODOS})[:;.,]?\d*($|\s.*$|[\{\[\(].+$)/g
```
simplified:
```regex
/\b(${TODOS})[:;.,]?\d*(?:\s|[\{\[\(].)?.*$/g
```

Check out this [Regex visualizer](https://regexper.com/) to verify, and let me know if you’d like me to submit a PR.
